src/olympia/blocklist/tests/test_admin.py

```diff
@@ -1361,7 +1361,7 @@ def test_signoff_approve(self):
         log_entry = LogEntry.objects.last()
         assert log_entry.user == user
         assert log_entry.object_id == str(mbs.id)
-        other_obj = addon_factory(id=mbs.id)
```

Comment:
That's an unrelated change that shouldn't be in this PR. Same below.

Reply:
hah, snap

Reply:
Removed